articles/reporting/health-score.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Health Score"
+description: "Learn about the Health Score, it's components, and recommendations on how to improve your sending."
+---
+
+Your Health Score is a predictive score to help you monitor your email health and identify problems before they surface in your normal metrics. Each day, the Health Score model predicts your overall engagement as compared with all senders using SparkPost, by analyzing dozens of different sending factors from your bounce rate to the relationship you have with your recipients. You'll receive a score between 0 and 100, informing you of your sending health. A good score is anything above 80.
+
+_**Note:** To provide the actionable insights and accurate analysis, the Health Score requires a minimum 1000 emails sent each day.  For any period where your sending does not pass these minimums, you will not have a Health Score._
```

Comment:
Should we also tell them that we need to have open data (lots of our larger customers don't use our engagement tracking)

Reply:
yeah, great call